src/components/Support.js

```diff
@@ -0,0 +1,164 @@
+/********************************************************************************
+ * Copyright (C) 2020 TypeFox and others.
```

Comment:
I assume we use typefox as a copyright since the file was copied from somewhere else?
Same for `support.js`.